frontend/packages/ceph-storage-plugin/src/components/converged-credentials/credentials.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { k8sGet } from '@console/internal/module/k8s';
+import { SecretModel, ConfigMapModel } from '@console/internal/models';
+import { createModalLauncher, ModalComponentProps } from '@console/internal/components/factory';
+import { OCS_NS } from '../../constants';
```

Comment:
We can use this: 
https://github.com/openshift/console/pull/4222/files#diff-4726b0cc864823cd33d5f323247c5bcbR5

Reply:
ack